packages/medusa/src/types/inventory.ts

```diff
@@ -101,6 +102,12 @@ export type UpdateInventoryLevelInput = {
   incoming_quantity?: number
 }
 
+export type UpdateReservationItemInput = {
+  quantity?: number
+  location_id?: string
```

Comment:
I think we can remove the location_id from this Input.

Reply:
Hmm, I think keeping it makes more sense to allow moving reservations from location to location, but I might be wrong. In our initial implementation it will not matter but if somebody wants to limit their stock based on location_id it would make sense right? 

I dont mind removing it, just sharing my thoughts 😄